src/Variable.cpp

```diff
@@ -474,10 +474,7 @@ Variable::~Variable(void)
 	for(i = field_vars.begin(); i != field_vars.end(); ++i)
 		delete (*i);
 	field_vars.clear();
-	if (init) {
-		delete init;
-		init = NULL;
```

Comment:
in a patch like this, we do not want changes that are unrelated to the task at hand, so do not remove assignments like this

Reply:
Would you like to get another bug report for such an implementation detail?